### PR TITLE
feat: use `mini-css-extract-plugin` for css chunking

### LINKED ISSUE #3: Generate a css file for each split point
_From @satazor on February 8, 2017 15:32_

Due to a limitation in [extract-text-plugin](https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/332) we are generating a single app css file. Ideally we would have a css file per webpack split point (per bundle). We must wait for this issue to be solved.

Alternatively we can explore other solutions such as https://github.com/kriasoft/isomorphic-style-loader

_Copied from original issue: moxystudio/react-with-moxy#2_

### PR DESCRIPTION
**Pull request by satazor**
By using `mini-css-extract-plugin` we finally have a css file per chunk.

The external-svg-sprite-loader package had to be updated because beta.4 is using `mini-css-extract-plugin` already.

Unrelated changes:
- Introduced `buildDir` and `buildUrlPath` in the constants to simplify code
- Renamed `stats` to `statsJson` in manifest to be more clear
- Added icons via CSS to test if `mini-css-extract-plugin` is playing well with SVGs

Closes #3.

### CODE REVIEW

File: scripts/build-manifest/create.js [hunk 1]
```diff
@@ -3,15 +3,14 @@
 const path = require('path');
 const sortChunks = require('webpack-sort-chunks').default;
 const escapeRegExp = require('lodash/escapeRegExp');
-const castArray = require('lodash/castArray');
 const merge = require('deepmerge');
 
-function parseWebpackStats(stats) {
-    const { hash, publicPath } = stats;
+function parseWebpackStats(statsJson) {
+    const { hash, publicPath } = statsJson;
```
Comment (antoniocapelo) [inline scripts/build-manifest/create.js]: as you're extracting `hash` and `publicPath` here, maybe also extract `chunks` and use it on line 13?

File: scripts/build-manifest/create.js [hunk 2]
```diff
@@ -21,7 +20,7 @@ function parseWebpackStats(stats) {
             assets[key] = assets[key] || [];
             assets[key].push({
                 file,
-                url: stats.publicPath + file,
+                url: statsJson.publicPath + file,
```
Comment (antoniocapelo) [inline scripts/build-manifest/create.js]: we can just use `publicPath`

File: scripts/build-manifest/create.js (hunk 1 above)
Comment (satazor) [reply scripts/build-manifest/create.js]: Done!

File: scripts/build-manifest/create.js (hunk 2 above)
Comment (satazor) [reply scripts/build-manifest/create.js]: Done!

**Pull request merged**